cypress/integration/expiredToken.spec.js

```diff
@@ -0,0 +1,26 @@
+const autoRecord = require('cypress-autorecord');
+
+describe('Expired Login', () => {
+  autoRecord();
+
+  beforeEach(() => {
+    cy.login()
+  })
+
+  it('refreshes auth token if token is invalid', function () {
```

Comment:
@rafayet-monon one small comment here, would it be better if we use an arrow function here? what do you think?
```suggestion
  it('refreshes auth token if token is invalid', () => {
```